modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -920,7 +922,7 @@ function Filters({fdsView, namespace}: IFDSViewSectionInterface) {
 						processClose();
 
 						const url = `${
-							item.type === 'date-time'
+							item.type === 'date' || item.type === 'date-time'
```

Comment:
If we focus on `date`  fields for the moment, keeping `date-time` here may be confusing for out future selves. It does not hurt anyways